gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/task/AbstractJKubeTask.java

```diff
@@ -75,12 +75,8 @@ public final void runTask() {
     jKubeServiceHub = initJKubeServiceHubBuilder().build();
     ImageConfigResolver imageConfigResolver = new ImageConfigResolver();
     try {
-      resolvedImages = initImageConfiguration(
-          kubernetesExtension.getApiVersion().getOrNull(),
-          getBuildTimestamp(null, null, javaProject.getBuildDirectory().getAbsolutePath(), DOCKER_BUILD_TIMESTAMP),
-          javaProject, kubernetesExtension.images, imageConfigResolver, kitLogger,
-          kubernetesExtension.getFilter().getOrNull(),
-          this::customizeConfig);
+      resolveImages(imageConfigResolver);
```

Comment:
This is not reusable as it relies on changes of state within the method.
The idea was to have a resolveImages method (just like you did), but that returned the list of images, so
```suggestion
      resolvedImages = resolveImages(imageConfigResolver);
```